src/components/movie-list/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import './style.less'
+import { Movie } from '../../models/movieDay';
+import MovieCard from '../movie-card';
+
+
+interface State {
+}
+
+interface Props {
+    movieList: Array<Movie>
+ }
+
+
+class MovieList extends React.Component<Props, State> {
```

Comment:
Component doesn't use state, no lifecycle methods, no functions. A good chance to practice functional components!:)